contrib/utilities/indent

```diff
@@ -30,6 +30,11 @@
 # from a build directory.
 #
 
+# Add the location 'setup_astyle.sh' installs astyle to to the local PATH.
+THIS_PATH="$(dirname $0)"              # relative
+THIS_PATH="$( cd $THIS_PATH && pwd )"  # absolutized and normalized
```

Comment:
One nit-picky thing: Can't this be done with `realpath`?

Reply:
Yes, thanks for pointing me to it.